espresso/docker-compose.yml

```diff
@@ -130,20 +389,25 @@ services:
         condition: service_started
       espresso-dev-node:
         condition: service_started
+      l2-genesis:
+        condition: service_completed_successfully
     environment:
-      L1_RPC: ${ESPRESSO_L1_PROVIDER}
-      OP_BATCHER_L1_ETH_RPC: ${ESPRESSO_L1_PROVIDER}
-      OP_BATCHER_L2_ETH_RPC: ${ESPRESSO_GETH_PROVIDER}
-      OP_BATCHER_ROLLUP_RPC: ${ESPRESSO_ROLLUP_PROVIDER}
-      ESPRESSO_URL: ${ESPRESSO_URL}
+      L1_RPC: http://l1-geth:${L1_HTTP_PORT}
+      OP_BATCHER_L1_ETH_RPC: http://l1-geth:${L1_HTTP_PORT}
+      OP_BATCHER_L2_ETH_RPC: http://op-geth:${OP_HTTP_PORT}
+      OP_BATCHER_ROLLUP_RPC: http://op-node-sequencer:${ROLLUP_PORT}
+      OP_BATCHER_ESPRESSO_URL: http://espresso-dev-node:${ESPRESSO_SEQUENCER_API_PORT},http://espresso-dev-node:${ESPRESSO_SEQUENCER_API_PORT}
     volumes:
       - ../packages/contracts-bedrock/lib/superchain-registry/ops/testdata/monorepo:/config
     command:
       - op-batcher
       - --espresso-light-client-addr=0x703848f4c85f18e3acd8196c8ec91eb0b7bd0797
       - --testing-espresso-batcher-private-key=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80 # Default value for testing
-      - --mnemonic=test test test test test test test test test test test junk                                    # Arbitrary value for testing
-      - --hd-path=m/44'/60'/0'/0/0                                                                                # Arbitrary value for testing
+      - --mnemonic=test test test test test test test test test test test junk # Arbitrary value for testing
+      - --hd-path=m/44'/60'/0'/0/0 # Arbitrary value for testing
+      - --throttle-threshold=0
+      - --max-channel-duration=1
```

Comment:
These last two options are a workaround for https://app.asana.com/1/1208976916964769/project/1209392461754458/task/1210958907358367